packages/http-server/src/__tests__/server.oas2.spec.ts

```diff
@@ -54,6 +73,24 @@ describe('server', () => {
     expect(response.statusCode).toBe(500);
   });
 
+  test('should not mock a request that is missing the required query parameters with no default', async () => {
+    const response = await server.fastify.inject({
+      method: 'GET',
+      url: '/pet/findByTags',
+    });
+
+    expect(response.statusCode).toBe(400);
+  });
+
+  test.skip('should automagically provide the parameters when not provided in the query string and a default is defined', async () => {
```

Comment:
@philsturgeon @chris-miaskowski 

Right now Prism does not use the `default` parameter value in case it's defined in the OAS document, so this test will fail. We need to decide whether this is something we want to support or not.

Moreover, it does not check for the `collectionFormat` property in the parameter, so that's another validation that's currently not happening.

Reply:
Yes when a specific code is requested and that code is not defined we can look for a body in `default`. @chris-miaskowski and I both got to agreeing on that.

Reply:
Can you make another JIRA issue for collectionFormat?

Reply:
SO-105 created.